spec/support/mockPartnerConfig.json

```diff
@@ -1,27 +1,21 @@
 {
-    "timeout": 1000,
+    "clientID": 0,
     "xSlots": {
-        "1": {
-            "placementId": "54321",
+        "xSlot1": {
+            "divID": "leaderboard1",
             "sizes": [
```

Comment:
It doesn't look like the code in `optimera-htb.js` uses `sizes` from these objects, so it can be removed.